docs/datasource_api_userguide.md

```diff
@@ -242,73 +260,82 @@ CREATE TABLE tpch_test.TARGET_TABLE_CUSTOMER (
 | spark.tispark.tidb.port     | tidb.port     | true     | TiDB Port                                                                                                   | -       |
 | spark.tispark.tidb.user     | tidb.user     | true     | TiDB User                                                                                                   | -       |
 | spark.tispark.tidb.password | tidb.password | true     | TiDB Password                                                                                               | -       |
 | database                    | -             | true     | TiDB Database                                                                                               | -       |
 | table                       | -             | true     | TiDB Table                                                                                                  | -       |
-| skipCommitSecondaryKey      | -             | false    | skip commit secondary key                                                                                   | false   |
-| enableRegionSplit           | -             | false    | do region split to avoid hot region during insertion                                                        | true    |
-| regionSplitNum              | -             | false    | user defined region split number during insertion                                                           | 0       |
-| replace                     | -             | false    | define the behavior of append.                                                                              | false   |
-| lockTTLSeconds              | -             | false    | tikv lock ttl, write duration must < `lockTTLSeconds`, otherwise write may fail because of gc               | 3600    |
-| writeConcurrency            | -             | false    | maximum number of threads writing data to tikv, suggest `writeConcurrency` <= 8 * `number of TiKV instance` | 0       |
+| skipCommitSecondaryKey      | -             | false    | To skip commit secondary key                                                                                   | false   |
+| enableRegionSplit           | -             | false    | To split Region to avoid hot Region during insertion                                                        | true    |
+| regionSplitNum              | -             | false    | The Region split number defined by user during insertion                                                           | 0       |
+| replace                     | -             | false    | To define the behavior of append                                                                              | false   |
+| lockTTLSeconds              | -             | false    | tikv lock ttl. The write duration must be longer than `lockTTLSeconds`, otherwise write might fail because of the Garbage Collection (GC).              | 3600    |
```

Comment:
```suggestion
| lockTTLSeconds              | -             | false    | TiKV's lock TTL. The write duration must be no longer than `lockTTLSeconds`, otherwise write might fail because of the Garbage Collection (GC).              | 3600    |
```